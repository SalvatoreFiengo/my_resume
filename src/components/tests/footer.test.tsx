import { shallow } from '../../enzyme';
import Footer from '../footer/footer';

describe('footer', ()=>{
    it('renders the footer without crashing',()=>{
        const footer = shallow(<Footer/>);
        expect(footer.getElements()).toMatchSnapshot()
    })
})