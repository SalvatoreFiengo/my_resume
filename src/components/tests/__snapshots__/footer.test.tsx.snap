// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders the footer without crashing 1`] = `
Array [
  <styled.div>
    <div
      className="attributions"
    >
      <p
        className="p-0 m-0"
      >
        <strong>
          Attributions:
        </strong>
      </p>
      <p
        className="p-0 m-0"
      >
        <small>
          <a
            className="me-1"
            href="https://www.maxpixel.net/Rome-Italy-Colosseum-Roman-Coliseum-City-2030643"
          >
            Rome's photo from MaxPixel
          </a>
          /
          <a
            className="ms-1"
            href="https://creativecommons.org/publicdomain/zero/1.0/deed.en"
          >
            CC 010
          </a>
        </small>
      </p>
      <p
        className="p-0 m-0"
      >
        <small>
          <a
            className="me-1"
            href="https://freepngimg.com/png/17607-resume-png-image/icon"
          >
            Ico image from Freepngimg
          </a>
          /
          <a
            className="ms-1"
            href="https://creativecommons.org/licenses/by-nc/4.0/"
          >
            CC BY-NC 4.0
          </a>
        </small>
      </p>
    </div>
    <div
      className="d-flex text-center justify-content-center"
    >
      <a
        aria-label="Github link here"
        className="me-2 mt-3"
        href="https://github.com/SalvatoreFiengo/my_resume.git"
        rel="noreferrer"
        target="_blank"
      >
        <i
          aria-hidden="true"
          className="fa fa-2x fa-github"
        />
      </a>
      <a
        aria-label="Linkedin Profile link here"
        className="ms-2 mt-3"
        href="https://www.linkedin.com/in/salvatorefiengo"
        rel="noreferrer"
        target="_blank"
      >
        <i
          aria-hidden="true"
          className="fa fa-2x fa-linkedin-square"
        />
      </a>
    </div>
  </styled.div>,
]
`;
